llvm/lib/Analysis/TargetLibraryInfo.cpp

```diff
@@ -29,7 +29,7 @@ static cl::opt<TargetLibraryInfoImpl::VectorLibrary> ClVectorLibrary(
                           "Accelerate framework"),
                clEnumValN(TargetLibraryInfoImpl::DarwinLibSystemM,
                           "Darwin_libsystem_m", "Darwin libsystem_m"),
-               clEnumValN(TargetLibraryInfoImpl::LIBMVEC_X86, "LIBMVEC-X86",
+               clEnumValN(TargetLibraryInfoImpl::LIBMVEC, "LIBMVEC",
```

Comment:
I think this needs changes in clang too as "LIBMVEC" is currently mapped to "LIBMVEC-X86":
https://github.com/llvm/llvm-project/blob/7d01b85c2a0aa8bc91f731cfeb2a7b6885b8d04a/clang/lib/Driver/ToolChains/CommonArgs.cpp#L932-L937

Reply:
Hmm. There actually seems to be a bug in clang's checks as it's checking for "LIBMVEC-X86" before this remapping, it should actually output `err_drv_unsupported_opt_for_target` for LIBMVEC on AArch64. I think once you rename  all uses of `LIBMVEC-X86` to `LIBMVEC` in clang that error will start working.

Reply:
Yep, I've just noticed the same.

Reply:
Done.